src/rsvc/rpc.h

```diff
@@ -71,8 +71,10 @@ extern struct crt_proto_format rsvc_proto_fmt;
 	((uint64_t)		(sai_size)		CRT_VAR) \
 	((d_rank_list_t)	(sai_ranks)		CRT_PTR)
 
-#define DAOS_OSEQ_RSVC_START /* output fields */		 \
-	((int32_t)		(sao_rc)		CRT_VAR)
+#define DAOS_OSEQ_RSVC_START /* output fields (rc: err count) */ \
```

Comment:
(style)  Macros with complex values should be enclosed in parentheses

Reply:
false-positive for CaRT RPC changes